src/components/Dashboard/Content/index.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import DashboardContent from '.';
+
+describe('DashboardContent', () => {
+  const dataTestId = 'dashboard-content';
+  it('renders DashboardContent and its components', () => {
+    render(<DashboardContent data-test-id={dataTestId} />);
```

Comment:
if choosing the below change, we don't need to pass the props here
```suggestion
// import the DashboardDataTestIds and use to get the HTML element
  it('renders DashboardContent and its components', () => {
    render(<DashboardContent  />);
```